packages/core/upload/server/controllers/utils/find-entity-and-check-permissions.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const _ = require('lodash');
```

Comment:
Are you fine with bundling the whole `lodash` here, or should this be destructured?